README.md

```diff
@@ -6,14 +6,13 @@ acbuild is a command line utility to build and modify App Container Images
 
 ## Rationale
 
-Dockerfiles are powerful and feature useful concepts such as build layers, 
-controlled build environment. At the same time, they lack flexibility 
+Dockerfiles are powerful, and feature useful concepts such as build layers and
```

Comment:
Is it actually necessary to mention `Dockerfile`s at all? 

If so, is what you want to say actually "We wanted a flexible, extensible tool for manipulating ACIs, well-integrated with the Unix toolkit, and as powerful and simple as the `Dockerfile` mechanism."